contrib/pg_stat_statements/expected/pg_stat_statements.out

```diff
@@ -434,20 +455,25 @@ SELECT 1+1 AS "TWO";
 (1 row)
 
 RESET ROLE;
-SELECT query, calls, rows FROM pg_stat_statements ORDER BY query COLLATE "C";
-               query               | calls | rows 
------------------------------------+-------+------
- CREATE ROLE regress_stats_user1   |     1 |    0
- CREATE ROLE regress_stats_user2   |     1 |    0
- RESET ROLE                        |     2 |    0
- SELECT $1 AS "ONE"                |     1 |    1
- SELECT $1 AS "ONE"                |     1 |    1
- SELECT $1+$2 AS "TWO"             |     1 |    1
- SELECT $1+$2 AS "TWO"             |     1 |    1
- SELECT pg_stat_statements_reset() |     1 |    1
- SET ROLE regress_stats_user1      |     1 |    0
- SET ROLE regress_stats_user2      |     1 |    0
-(10 rows)
+SELECT count(*), query, sum(calls) calls, sum(rows) rows FROM pg_stat_statements GROUP BY query ORDER BY query COLLATE "C";
+ count |                              query                              | calls | rows 
+-------+-----------------------------------------------------------------+-------+------
+     1 | CREATE ROLE regress_stats_user1                                 |     1 |    0
+     3 | CREATE ROLE regress_stats_user1;                                |     3 |    0
+     1 | CREATE ROLE regress_stats_user2                                 |     1 |    0
+     3 | CREATE ROLE regress_stats_user2;                                |     3 |    0
+     1 | RESET ROLE                                                      |     2 |    0
+     3 | RESET role                                                      |     6 |    0
```

Comment:
Why do we have `ROLE` and `role` here?

Reply:
https://github.com/hughcapet/gpdb/blob/4845614483b68f72dc80e179c1e22c905140245c/src/backend/parser/gram.y#L2279-L2283

https://github.com/hughcapet/gpdb/blob/4845614483b68f72dc80e179c1e22c905140245c/src/backend/utils/misc/guc.c#L8592

shouldn't we force upper-case here...?